clang/test/CodeGen/scoped-fence-ops.c

```diff
@@ -0,0 +1,179 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 %s -emit-llvm -o - -triple=amdgcn-amd-amdhsa -ffreestanding \
+// RUN:   -fvisibility=hidden | FileCheck --check-prefix=AMDGCN %s
+//: %clang_cc1 %s -emit-llvm -o - -triple=spirv64-unknown-unknown -ffreestanding \
+//:   -fvisibility=hidden | FileCheck --check-prefix=SPIRV %s
```

Comment:
better to have a broader test, like arm, x86, etc.

Reply:
Those targets aren't interesting because their handling of scopes just work exactly like the normal fence at least as far as the IR is concerned.

Reply:
but at least show the front end can generate IR